src/popup/components/ErrorKeyRevoked.tsx

```diff
@@ -109,7 +126,11 @@ const MainScreen = ({
         <Button onClick={() => requestDisconnect()} loading={loading}>
           {t('keyRevoked_action_disconnect')}
         </Button>
-        <Button onClick={() => requestReconnect()}>
+        <Button
+          onClick={() => {
+            setScreen('consent-reconnect');
+          }}
```

Comment:
Nit: inline like this:
```suggestion
          onClick={() => setScreen('consent-reconnect')}
```